### DISCUSSION TITLE
Version number for each flavor

### DISCUSSION THREAD

**Discussion by mucahitbircan** [AngeloAvv/flutter_flavorizr#237] [Q&A]
Hi Flutter Community, I want to use a separate version number for each flavor. how can I do that

**Comment by AngeloAvv** [comment]
Hey @mucahitbircan , this is more like a native side thing than something that has to do with flutter_flavorizr. 

I encourage you to take a look at the native docs to better understand how to do that:
On the Android side: https://developer.android.com/studio/publish/versioning?hl=en

On the iOS side, I think you can take advantage of variables to accomplish it

**Reply by mucahitbircan** [reply]
Hey @AngeloAvv ,
I wanted to make different apps using one code base and automate everything. Looks like I got a bit lost there. Thanks for your help!
